entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ ! -z "$WAIT_TIME" ]; then
```

Comment:
For whatever reason, updating to the new ECS AMI started making Empire start a lot faster than the postgres container, which means it couldn't connect and would just get stuck in a restart loop. I've seen it happen previously, but seems to happen consistently with the new AMI.

I'm not super happy adding this. I think the two options we have are to do this, or remove the postgres container and replace it with an RDS instance. The only reason we haven't done that is that RDS takes a lot of time to create.